src/core/server/rendering/types.ts

```diff
@@ -24,6 +24,7 @@ export interface RenderingMetadata {
   i18n: typeof i18n.translate;
   locale: string;
   darkMode: boolean;
+  themeVersion?: string;
```

Comment:
Mainly because there's an import of the Font file that doesn't have access to the current theme version.  https://github.com/elastic/kibana/blob/ab2a80f4b0f830947e57245fcfe2961a11d7ecd3/x-pack/plugins/security/server/prompt_page.tsx#L67

Reply:
Hum, yea. Ideally we'd want to adapt these two calls from security to retrieve the themeVersion setting, however it wouldn't do much, as these pages are rendered when security is enabled and user not authenticated, meaning that it would default to `v8` anyway, so I guess having it optional and performing the default/fallback in core's component probably makes sense.